src/lib/ecore_win32/ecore_win32_cursor.c

```diff
@@ -2,8 +2,11 @@
 # include <config.h>
 #endif
 
-#define WIN32_LEAN_AND_MEAN
+#ifndef WIN32_LEAN_AND_MEAN
+# define WIN32_LEAN_AND_MEAN
+#endif
 #include <windows.h>
+#include <windowsx.h>
```

Comment:
Why now we need `windowsx.h` here?

Reply:
oops, actually it uses `winuser.h`, not `windowsx.h`.
```suggestion
#include <winuser.h>
```

Reply:
hummm, and why we need `winuser.h`?

Reply:
I mean, it seems that it wasn't needed before, so why it is needed now?

Reply:
There are several functions from `winuser.h` been used here, maybe `windows.h` includes it already?